mods/doors/init.lua

```diff
@@ -1,435 +1,440 @@
-doors = {}
 
--- Registers a door
-function doors.register_door(name, def)
-	def.groups.not_in_creative_inventory = 1
+--[[
 
-	local box = {{-0.5, -0.5, -0.5, 0.5, 0.5, -0.5+1.5/16}}
+Copyright (C) 2012 PilzAdam
+  modified by BlockMen (added sounds, glassdoors[glass, obsidian glass], trapdoor)
+Copyright (C) 2015 - Auke Kok <sofar@foo-projects.org>
 
-	if not def.node_box_bottom then
-		def.node_box_bottom = box
-	end
-	if not def.node_box_top then
-		def.node_box_top = box
-	end
-	if not def.selection_box_bottom then
-		def.selection_box_bottom= box
+--]]
+
+-- our API object
+doors = {}
+
+-- private data
+_doors = {}
+_doors.registered_doors = {}
+_doors.registered_trapdoors = {}
+
+-- returns an object to a door object or nil
+doors.get = function(pos)
```

Comment:
I guess it's like the eternal war about where to put the braces in C-like languages.

This is just personal opinion, but I (generally) use `function foo.bar()` when I mean to use `foo` as a namespace, while using `foo.bar = function()` if `foo` is an object/record/whatever. Sort of implicit documentation or something.

Did not mean that comment as "do it or else..."; as I said, it was "minor nitpicking". Either way, thank you for making it consistent.